JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -21,9 +21,11 @@ import { FileObj, UploadStatus } from '../../types/FileUpload'
 import Button from '../shared/Button'
 import './Import.css'
 import UploadFileListItem from './UploadFileListItem'
+import { formatBytesSize, megabytesToBytes } from 'utils/formatters'
 
-const MAX_UPLOAD_FILE_SIZE_IN_BYTES =
-  process.env.REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_BYTES ?? 128 * 1024 * 1024
+const MAX_UPLOAD_FILE_SIZE_IN_BYTES = megabytesToBytes(
+  +(process.env.REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_MEGABYTES ?? 128)
```

Comment:
Don't you need to convert `process.env.REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_MEGABYTES` to a number first?

Reply:
` +(process` the "+" operator did the conversion .

Reply:
Ah nice! Had missed that 👍🏽